README.md

```diff
@@ -1,46 +1,15 @@
-# Getting Started with Create React App
+Title — What is your app/project called?
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+Overview — With the increased use of online classes, the need for a class management system is undeniable. The goal of this class management system is to simplify processes, keep things organized, and provide insights acurately & visually.
 
-## Available Scripts
+Resouces:
 
-In the project directory, you can run:
+  -Design: https://www.figma.com/file/ojDUYgb1gmWJ9BTEVmdebp/Class-Management-%2F%2F-Chingu?node-id=0%3A1
+  
+  -Visual Sitemap: https://octopus.do/wpf8xbccze
+  
+Features — What are some key things your project can do?
 
-### `npm start`
```

Comment:
I think `npm install` and `npm start` should still be in the README. Just some instruction on how to run the project locally